src/AKCore/DataModel/AKContext.cs

```diff
@@ -221,6 +221,7 @@ public class Event
     public string Stand { get; set; }
     public List<SignUp> SignUps { get; set; }
     public bool Secret { get; set; }
+    public bool AllowsSignUps { get; set; }
```

Comment:
Since this is about Disabling signups, should we perhaps have that as the database prop to avoid to many nagations?

Reply:
Im confused. Do you mean to change the prop to IsDisabled for example?

Reply:
Yes, unless im maybe missing something about what its meant to do.

Or just call it `Disabled`.

Reply:
Yes. I can do that. Good point.

Reply:
Did the changes now but do you have a good way for reverting the old migration? Right now i have two and a rename, which i guess will not work for other setups.

![Skärmbild 2024-09-07 110222](https://github.com/user-attachments/assets/4c3417ff-f906-41f8-bd4f-099989644a7b)

Reply:
Fixed now